.vitepress/theme/index.ts

```diff
@@ -24,6 +27,5 @@ export default Object.assign({}, VPTheme, {
     app.provide('prefer-composition', preferComposition)
     app.provide('prefer-sfc', preferSFC)
     app.provide('filter-headers', filterHeadersByPreference)
-    app.component('VueSchoolLink', VueSchoolLink)
```

Comment:
FYI this should not be removed as this component is needed in many pages. Fixed in 5281fd3